playbooks/byo/openshift-cluster/config.yml

```diff
@@ -7,5 +7,4 @@
   vars:
     openshift_cluster_id: "{{ cluster_id | default('default') }}"
     openshift_debug_level: "{{ debug_level | default(2) }}"
-    openshift_deployment_type: "{{ deployment_type }}"
     openshift_deployment_subtype: "{{ deployment_subtype | default(none) }}"
```

Comment:
Rarely used, only ever set to `None` or `'registry'`. According to the changelog in openshift-ansible.spec:

> Secure registry for atomic registry deployment (deployment_subtype=registry)